tools/alpine/Dockerfile

```diff
@@ -16,6 +16,10 @@ RUN cat /tmp/packages.$(uname -m) >> /tmp/packages && \
 # add the tools for WireGuard, since the kernel module is now included, but from edge/testing
 RUN apk fetch --recursive -o /mirror/$(uname -m) -X http://dl-cdn.alpinelinux.org/alpine/edge/testing -U wireguard-tools
 
+# add OVMF for EFI boot (x86 only)
+RUN [ $(uname -m) = x86_64 ] && \
+    apk fetch --recursive -o /mirror/$(uname -m) -X http://dl-cdn.alpinelinux.org/alpine/edge/community -U ovmf
```

Comment:
Perhaps we should do
```
apk fetch --recursive -o /mirror/$(uname -m) -X http://dl-cdn.alpinelinux.org/alpine/edge/community -U $(cat packages.community)
```
and list them there instead of in the main `packages`, so at least we are explicit about what we are willing to take from community?